modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/WizardPage.spec.tsx

```diff
@@ -0,0 +1,269 @@
+			'.multi-step-indicator-label'
+		);
+
+		const buttonConnect = getByRole('button', {name: /1/i});
+
+		const buttonProperty = getByRole('button', {name: /2/i});
+
+		const buttonPeople = getByRole('button', {name: /3/i});
+
+		const buttonAttributes = getByRole('button', {name: /4/i});
+
+		expect(titleMultiSteps[0]).toHaveTextContent('connect');
+
+		expect(titleMultiSteps[1]).toHaveTextContent('property');
+
+		expect(titleMultiSteps[2]).toHaveTextContent('people');
+
+		expect(titleMultiSteps[3]).toHaveTextContent('attributes');
+
+		expect(buttonConnect).toBeInTheDocument();
+
+		expect(buttonProperty).toBeInTheDocument();
+
+		expect(buttonPeople).toBeInTheDocument();
+
+		expect(buttonAttributes).toBeInTheDocument();
+	});
+
+	it('not possible go to others steps without connect token', () => {
+		const {getByRole} = render(<WizardPage />);
+
+		const buttonConnect = getByRole('button', {name: /1/i});
+
+		const buttonProperty = getByRole('button', {name: /2/i});
+
+		const buttonPeople = getByRole('button', {name: /3/i});
+
+		const buttonAttributes = getByRole('button', {name: /4/i});
+
+		expect(buttonConnect).not.toHaveClass('active');
+
+		expect(buttonProperty).not.toHaveClass('active');
+
+		expect(buttonPeople).not.toHaveClass('active');
+
+		expect(buttonAttributes).not.toHaveClass('active');
+	});
+
+	it('render property step with token connection', async () => {
+		await act(async () => {
+			const {getByRole, getByText} = render(<WizardPage />);
+
+			fetch.mockResponse(JSON.stringify(responseTokenConnect));
+
+			fireEvent.click(getByRole('button', {name: /2/i}));
+
+			render(
+				<PropertyStep onCancel={() => {}} onChangeStep={() => {}} />
+			);
+
+			const propertyStepDescription = getByText('property-description');
+
+			const propertyStepTitle = getByText('property-assignment');
+
+			expect(propertyStepDescription).toBeInTheDocument();
+
+			expect(propertyStepTitle).toBeInTheDocument();
+		});
+	});
+
+	it('after the property step, render people step with token connection', async () => {
+		await act(async () => {
+			fetch.mockResponseOnce(JSON.stringify(responseTokenConnect));
+
+			render(
+				<PropertyStep onCancel={() => {}} onChangeStep={() => {}} />
+			);
+
+			fireEvent.click(screen.getByRole('button', {name: /next/i}));
+
+			fetch.mockResponseOnce(JSON.stringify(responsePeopleMock));
+
+			const {getByText} = render(
+				<PeopleStep onCancel={() => {}} onChangeStep={() => {}} />
+			);
+
+			const peopleStepTitle = getByText('sync-people');
+
+			expect(peopleStepTitle).toBeInTheDocument();
+		});
+	});
+
+	it('after the people step, render attributes step with token connection', async () => {
+		await act(async () => {
+			fetch.mockResponseOnce(JSON.stringify(responseTokenConnect));
+
+			const {container, getByText} = render(
+				<PropertyStep onCancel={() => {}} onChangeStep={() => {}} />
+			);
+
+			const buttonNext = container.querySelectorAll('.btn');
```

Comment:
Since you're selecting all the buttons, please rename this const to `buttonsNext` (plural) to make it more clear.

Reply:
Done!